lib/gssapi/mech/gss_mech_switch.c

```diff
@@ -314,14 +314,6 @@ _gss_load_mech(void)
 		if (found)
 			continue;
 
-#ifndef RTLD_LOCAL
```

Comment:
You should keep these.  Just because we don't always get them doesn't mean we shouldn't try to.

Reply:
Oh, never mind.